src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class FruitTransaction {
+    private OperationsWithFruits operation;
+    private String fruitName;
+    private int quantity;
```

Comment:
As you've decided to decouple `Fruit` itself from the `FruitTransaction`, consider using `Fruit` here instead of `fruitName` or just remove `Fruit` class.